suite-native/module-accounts-import/src/components/SelectableNetworkItem.tsx

```diff
@@ -48,7 +51,7 @@ export const SelectableNetworkItem = ({ symbol, onPress }: SelectableAssetItemPr
                         <Text variant="body">{networkName}</Text>
                         <HStack alignItems="center" justifyContent="center">
                             <Text variant="hint" color="textSubdued">
-                                <NetworkSymbolFormatter value={symbol} />
+                                {formattedNetworkSymbol}
```

Comment:
Why you didn't used component? You can pass params as props too.

Reply:
awesome, fixed by a653fd8b81